include/wx/cmndata.h

```diff
@@ -222,6 +244,9 @@ class WXDLLIMPEXP_CORE wxPrintDialogData: public wxObject
     bool            m_printEnablePrintToFile = true;
     wxPrintData     m_printData;
 
+    int m_maxPageRanges = 64;
+    wxVector<wxPrintPageRange> m_printPageRanges;
```

Comment:
Let's use the standard class directly in the new code:
```suggestion
    std::vector<wxPrintPageRange> m_printPageRanges;
```